src/components/UIShell/SideNav.js

```diff
@@ -64,24 +84,27 @@ export default class SideNav extends React.Component {
       className: customClassName,
       translateById: t,
     } = this.props;
-    const { isExpanded } = this.state;
+    const { isExpanded, isFocused } = this.state;
     const accessibilityLabel = {
       'aria-label': ariaLabel,
       'aria-labelledby': ariaLabelledBy,
     };
-    const assistiveText = isExpanded
-      ? t('carbon.sidenav.state.open')
-      : t('carbon.sidenav.state.closed');
+    const assistiveText =
+      isExpanded || isFocused
+        ? t('carbon.sidenav.state.open')
+        : t('carbon.sidenav.state.closed');
     const className = cx({
       [`${prefix}--side-nav`]: true,
-      [`${prefix}--side-nav--expanded`]: isExpanded,
+      [`${prefix}--side-nav--expanded`]: isExpanded || isFocused,
```

Comment:
@asudoh @vpicone I did change the PR to remove the need for an additional className and styles. Is this preferable?

Reply:
Good call :)